src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -1094,17 +1066,26 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     }
   }
 
+  private def sendInitialStartMessage(callKey: CallKey) = {
```

Comment:
not sure this is necessary, but if it is it should be used everywhere it's applicable and one-lined